earth_enterprise/BUILD.md

```diff
@@ -0,0 +1,58 @@
+If you want to build, install, or run Portable server, see the [Portable Server page](https://github.com/google/earthenterprise/wiki/Portable-Server).
+
+# Building Earth Enterprise Fusion and Server
+
+Building is currently supported on 64-bit versions of Ubuntu 14.04 LTS, Ubuntu 16.04 LTS, RHEL 6, RHEL 7, and CentOS 7.
+
+
+## GEE 5.2.1 Build Prerequisites (All platforms) 
+
+1. Setup required Tools and Dependencies
+    * git 1.7.1+
+    * git lfs 
+    * gcc 4.8.x
+    * scons 2.0.x+
+    * python 2.6.x+
+    
+    Different operating systems have different means to setup up these dependencies. For Linux build environments, see either the [Redhat and Centos Setup Instructions](./BUILD_RHEL_Centos.md)  
+or the [Ubuntu Setup Instructions](./BUILD_RHEL_Centos.md) for those specific platforms on how to setup the dependencies, tools and compilers. 
+ 
+1. Clone the EarthEnterprise repository in your build environment
+    
+    ___Note: for development you should follow the instructions on the [Git Contributions](https://github.com/google/earthenterprise/wiki/Git-Contributions) page to clone the GEE repo to your personal fork, so that you can submit git pull-requests back to this repo.___
+           
+    * Method 1: Clone and download LFS files in one step (may be slower and more error prone).
```

Comment:
I've always thought that it would be a good idea to reverse these methods or just eliminate the less reliable one. Thoughts?

Reply:
Personally I like the first method better and I haven't had many reliability issues with it.

Reply:
I've had method one fail a couple of times, but always seems to work out on a retry.   What I do like about method one is that there are less external dependencies.  Two votes for it to stay the same?